### ISSUE TITLE
Sprint 61 - Facilities

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#9523]
<details><summary>Background</summary>

🥅   — Sprint goals (pending Sprint planning)
💡 — Design
💧  — Drupal engineering
🎉 — FE
🎓  — Training / Change management / KB
📟 — Helpdesk
🔮  —  Planning / roadmap / epic refinement
📊  —  Analytics
🚰  — Service design
🌱 — Good first task

</details>

## #9096 Facility COVID status post-launch

- [ ]  #9522 🥅

## Facility Locator

- [ ] Epic #9486
  - [ ] #9485
  - [ ] other candidates:
    - [ ] #9501 
    - [ ] #9503 
    - [ ] #9502 
- [ ] #8769 (David, possibly with a designer)

Refine
- [ ] Bounding box to Radius got rolled back to bounding box, possibly low priority to go back to radius.

</details>

## Q3 planning

- [ ] Thursday: TBD
- [ ] 

## VAMC

- [ ] #9220 (FE prod defect, Billing and insurance)

###  #3151 3 VAMC Top task pages

Support for ongoing change management.

#### Research

- [ ] #9071 
  - [ ] stretch: #9072 

<details><summary>CMS (on pause until design is validated)</summary>

- [x] #7626 (sprint 55)
- [ ] Create proof of concept
- [ ] CMS design discovery epic
  - [ ] #8863
- [ ] Refine epic #1939 (for DoD facilities)
  - [ ] #1940 
  - [ ] #1941 
  - [ ] #1942 
  - [ ] #1943  

</details> 

## Vet Center CAP geodata

- [ ] #8627
- [ ] #8733 (not sure if this is required)

##  #9318 Hours

- [ ] #9358 (work done, value delivered, but script not yet merged for future re-use.)
- [ ] #9390 (PR ready)
- [ ] #8302 
  - [ ] #9357
- [ ] Others forthcoming, possibly. 

## Events V2 (Max)

- [ ] #9456 
- [ ] #9455 

## Lighthouse / Facility API data

<details><summary>Blocked by V1</summary>

- [ ] #8017  
- [ ] #4769 

</details>

## Refinement

Thursday will be focused on Q3 planning

Not Thursday
- [ ] #7707 and #9286 
- [ ] Geodata for Vet Center CAPs 
- [ ] Q3 planning
